packages/material-ui/src/NoSsr/index.d.ts

```diff
@@ -1,2 +1 @@
-export { default } from './NoSsr';
-export * from './NoSsr';
+export { default } from '@material-ui/unstyled/NoSsr';
```

Comment:
Shouldn't we also export the `NoSsrProps`? Otherwise, it is a breaking change.

Reply:
Indeed we should 🙂